### ISSUE TITLE
SRCDS freeze before starting default map nmo_chinatown

### ISSUE THREAD

**Issue by Lunreth** [nmrih/source-game#1419]
Tested on a brand new server using SteamCMD (without sourcemod or any custom content) - Windows

![srcds_freeze](https://user-images.githubusercontent.com/26940632/219486862-99b53d5c-f318-4bef-a76a-e480216b710e.jpg)

Something about 1.13.2 broke private servers, not paid ones

**Comment by Lunreth** [comment]
Also using "app_update 317670" and "app_update 317670 validate" won't fix anything, files are all valid according to SteamCMD

**Comment by Lunreth** [comment]
Console throws "NUM_FOR_EDICT: bad pointer" before starting nmo_chinatown

**Comment by felis-catus** [comment]
Likely caused by #1417, closing as this focuses on particular server configuration and we're tracking the possible issue elsewhere.

**Comment by Lunreth** [comment]
It has nothing to do with maphacks, i tried removing them and same error happens
